src/plugins/unified_doc_viewer/public/hooks/use_es_doc_search.ts

```diff
@@ -131,7 +131,7 @@ export function buildSearchBody(
           filter: [{ ids: { values: [id] } }, { term: { _index: index } }],
         },
       },
-      stored_fields: computedFields.storedFields,
+      stored_fields: ['*'],
```

Comment:
Is this necessary with the change to `SearchSource`, or can we remove this line altogether now?